### PR TITLE
YOMA-503 - refactor(UserChallenges): merge MyChallenges and UserChallenges modules.

### PR DESCRIPTION
**Pull request by cameronolivier**
## Scope [YOMA-503](https://yomaworld.atlassian.net/browse/YOMA-503)
- consolidating all User Challenge credentials UI and data management into a single module

## Work Done
- merged MyChallenges with UserChallenges
- renamed NewChallengesForm and flattened into UserChallenges
- slight refactoring on CvCard - pulled EditButton into its own folder.
- updated HomeNavigation to "UserChallenges".

## Steps to Test
- navigating around the app should still work

### CODE REVIEW

File: src/modules/UserChallenges/UserChallenges.middleware.test.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import * as UserActions from '../User/User.reducer'
 import * as SUT from './UserChallenges.middleware'
 import { getUserChallengesSuccess, normaliseUserChallengesSuccess, setUserChallenges } from './UserChallenges.reducer'
 
-describe('modules/UserChallenges/UserChallenges.middleware', () => {
+describe('modules/Challenges/Challenges.middleware', () => {
```
Comment (sciphergfx) [inline src/modules/UserChallenges/UserChallenges.middleware.test.ts]: Not super crucial, but Can we change Challenges to match the wording UserChallenges?

Comment (cameronolivier) [reply src/modules/UserChallenges/UserChallenges.middleware.test.ts]: yes - I think that was an auto-renaming...

File: release-notes.txt [hunk 2]
```diff
@@ -1,8 +1,9 @@
 Ver 0.0.1
```
Comment (sciphergfx) [inline release-notes.txt:1]: there's a conflict here

File: src/components/CvCard/index.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import CvCard from './CvCard'
-import CvCardListBody from './CvCardListBody'
+import CvCardListBody from './CvCardListBody/CvCardListBody'
```
Comment (sciphergfx) [inline src/components/CvCard/index.ts]: suggestion (non-blocking): CvCardListBody is already imported via the index file so './CvCardListBody'?

Comment (cameronolivier) [reply src/components/CvCard/index.ts]: yeah. I'm not sure why it auto-corrected there. I'll adjust, and then if you can approve, presuming you're happy with it.

Comment (cameronolivier) [reply src/components/CvCard/index.ts]: @Sheyie Bowhalle  - updated and ready for approval.

**Review by sciphergfx — APPROVED** [review]
All good

**Pull request merged**